web/css/source/variables/overrides/_blank.less

```diff
@@ -1,2 +1,4 @@
 //  Form field note icon
 @form-field-note-icon-font__content: false;
+//  Icon text
+@_icon-font-text-hide: false;
```

Comment:
looks like this variable could be used only as a private variable for a specific mixin like .lib-icon. 
it does nothing. has to be removed

Reply:
Sorry forgot to delete that code before pushing to Github